src/picomsg/src/oper_db_writer.h

```diff
@@ -3,17 +3,21 @@
 
 #include <stdint.h>
 #include "terminator_conn/c_api/conn.h"
-
+#include "oper_data_struct.h"
```

Comment:
Keep empty line between includes and any other piece of code (in this case the `#ifdef`)